pkg/profiler/cpu/cpu.go

```diff
@@ -415,6 +417,25 @@ func (p *CPU) ensureUnwindTables(pid int) error {
 			}
 		}
 
+		if compact {
+			compact := unwind.PlanTable{}
+			prevCFARegister := uint64(0)
+			prevCFAOffset := int64(0)
+			prevRBPRegisterOffset := int64(0)
+
+			for _, row := range pt {
+				newEntry := row.CFA.Reg != prevCFARegister || row.CFA.Offset != prevCFAOffset || row.RBP.Offset != prevRBPRegisterOffset
```

Comment:
```suggestion
				notSameAsPrev := row.CFA.Reg != prevCFARegister || row.CFA.Offset != prevCFAOffset || row.RBP.Offset != prevRBPRegisterOffset
```
I think this is semantically more correct, no?

Reply:
This has changed after I've commented above :see_no_evil:

Reply:
Was just changing it :D Thanks for noticing, the previous name was outright wrong!